.circleci/config.yml

```diff
@@ -35,107 +35,107 @@ commands:
 jobs:
   macOS10_15:
     macos:
-      xcode: "11.3.0"
+      xcode: "11.3.1"
     environment:
       DESTINATION: platform=macOS,arch=x86_64
       CIRCLE_XCODE_SCHEME: Apollo
-      CIRCLE_XCODE_SDK: macosx10.15
+      CIRCLE_XCODE_SDK: macosx10.15.1
```

Comment:
The SDKs shouldn't be changed here - only the version of Xcode has been bumped, not the SDK itself. This is why the builds are all failing. 

Here's a screenshot from my machine of the SDKs included with 11.3.1: 
<img width="496" alt="Screen Shot 2020-01-31 at 9 47 20 AM" src="https://user-images.githubusercontent.com/1976498/73561908-d1e68d80-440e-11ea-92ac-2b5461818970.png">